content/en/docs/setup/getting-started/index.md

```diff
@@ -327,12 +327,17 @@ $ export INGRESS_HOST=$(kubectl get po -l istio=ingressgateway -n istio-system -
 
 ### Verify external access {#confirm}
 
-Confirm that the Bookinfo application is accessible from outside. Copy the
-output of this command and paste into your browser:
+Confirm that the Bookinfo application is accessible from outside
+by viewing the Bookinfo product page using a browser.
 
-{{< text bash >}}
-$ echo http://$GATEWAY_URL/productpage
-{{< /text >}}
+1.  Run the following command to retrieve the external address of the Bookinfo application.
+
+    {{< text bash >}}
+    $ echo http://$GATEWAY_URL/productpage
+    http://192.168.99.100:32194/productpage
```

Comment:
I'm not sure whether adding example output here is helpful or not. I wonder if somebody will read the next step ("Paste the output from the previous command") and will then copy the displayed output above and try it. It's providing one more way of making a stupid mistake.

Reply:
I agree I think it's not useful and can cause a potential mistake from the user.